### PR TITLE
Update nuget packages and generate nuget package on build

### PR DESCRIPTION
**Pull request by jmarolf**

### CODE REVIEW

File: xunit.runner.wpf/packages.config [hunk 1]
```diff
@@ -1,18 +1,19 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
   <package id="CommonServiceLocator" version="1.3" targetFramework="net452" />
-  <package id="MvvmLight" version="5.1.1.0" targetFramework="net452" />
-  <package id="MvvmLightLibs" version="5.1.1.0" targetFramework="net452" />
-  <package id="System.Collections" version="4.0.0" targetFramework="net46" />
+  <package id="MvvmLight" version="5.3.0.0" targetFramework="net46" />
```
Comment (Pilchie) [inline xunit.runner.wpf/packages.config]: Who picked packages.config for this project.  Shouldn't we project.json all the things? :trollface:

File: xunit.runner.wpf/xunit.runner.wpf.nuspec [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  <metadata>
+    <id>xUnit.Runner.WPF</id>
+    <version>1.0.0</version>
+    <title>xUnit.Runner.WPF</title>
+    <authors>Pilchie</authors>
+    <owners>Pilchie</owners>
+    <projectUrl>https://github.com/Pilchie/xunit.runner.wpf</projectUrl>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <description>XUnit Gui written in WPF</description>
+    <releaseNotes>Summary of changes made in this release of the package.</releaseNotes>
```
Comment (Pilchie) [inline xunit.runner.wpf/xunit.runner.wpf.nuspec]: You might consider either omitting this or providing a better value.

File: xunit.runner.wpf/xunit.runner.wpf.nuspec [hunk 3]
```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0"?>
+<package >
+  <metadata>
+    <id>xUnit.Runner.WPF</id>
```
Comment (Pilchie) [inline xunit.runner.wpf/xunit.runner.wpf.nuspec]: How does Brad case the official package ids?

File: xunit.runner.wpf/xunit.runner.wpf.nuspec [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+    <owners>Pilchie</owners>
+    <projectUrl>https://github.com/Pilchie/xunit.runner.wpf</projectUrl>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <description>XUnit Gui written in WPF</description>
+    <releaseNotes>Summary of changes made in this release of the package.</releaseNotes>
+    <copyright>Copyright 2016</copyright>
+    <tags>XUnit Gui test runner</tags>
+  </metadata>
+  <files>
+    <file src="*.dll" target="tools\"/>
```
Comment (Pilchie) [inline xunit.runner.wpf/xunit.runner.wpf.nuspec]: What all gets included, and how big is this package?

Comment (jmarolf) [inline xunit.runner.wpf/xunit.runner.wpf.nuspec]: xUnit.Runner.WPF.1.0.0.nupkg size **487 KB**

xUnit.Runner.WPF.1.0.0.nupkg\tools\
* GalaSoft.MvvmLight.Extras.dll
* GalaSoft.MvvmLight.Platform.dll
* GalaSoft.MvvmLight.dll
* Microsoft.Practices.ServiceLocation.dll
* Microsoft.WindowsAPICodePack.Shell.dll
* Microsoft.WindowsAPICodePack.dll
* System.Collections.Immutable.dll
* System.Windows.Interactivity.dll
* xunit.abstractions.dll
* xunit.runner.data.dll
* xunit.runner.utility.desktop.dll
* xunit.runner.worker.exe
* xunit.runner.wpf.exe
* xunit.runner.wpf.exe.config

File: xunit.runner.wpf/xunit.runner.wpf.nuspec (hunk 3 above)
Comment (jmarolf) [inline xunit.runner.wpf/xunit.runner.wpf.nuspec]: brad uses all lower case for the id but uppercase for the title

Comment (Pilchie) [inline xunit.runner.wpf/xunit.runner.wpf.nuspec]: Can we match that here?

File: xunit.runner.wpf/xunit.runner.wpf.nuspec (hunk 4 above)
Comment (Pilchie) [inline xunit.runner.wpf/xunit.runner.wpf.nuspec]: Okay, that looks great.

**Comment by jmarolf** [comment]
@DustinCampbell do you have the power to merge this in?

**Pull request merged**